corehq/apps/smsbillables/management/commands/retrobill_india_unicel_03012016.py

```diff
@@ -0,0 +1,106 @@
+import datetime
+from decimal import Decimal
+from sys import stdout
+from corehq.apps.accounting.models import (
+    Currency,
+    BillingAccount,
+    CreditLine,
+    CreditAdjustmentReason
+)
+from corehq.apps.smsbillables.models import SmsBillable
+from django.core.management import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    help = 'Fix conversion issue for INR-based smsbillables on india server created' \
+           'prior to 3/1/2016'
+
+    def handle(self, *args, **options):
+        if raw_input(
+            'Are you sure you want to re-bill all SMS billables with'
+            ' gateway fees in INR calculated prior to March 1, 2016?\n'
+            'This action will invalidate the old billables, create new ones,'
+            ' and add the difference as general account credit to each'
+            ' affected domain. \n[y/n]'
+        ).lower() != 'y':
+            raise CommandError('abort')
+
+        inr = Currency.objects.filter(code="INR").first()
+
+        affected_criteria = SmsBillable.objects.filter(
+            gateway_fee__currency__code=inr.code,
+            date_created__lt=datetime.date(2016, 3, 1),
+            is_valid=True
+        )
+        for unique_b in affected_criteria.order_by('domain').distinct('domain'):
```

Comment:
no need to change now, but it would be a little cleaner to add `.values_list('domain', flat=True)` here so that we only iterate over the list of domains if that's all we're looking for